rm_vk/rm_vk/Common/Cell/MainTableViewCell.swift

```diff
@@ -0,0 +1,33 @@
+// MainTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Главная универсальная ячейка
+final class MainTableViewCell: UITableViewCell {
+    // MARK: - IBOutlet
+
+    @IBOutlet private var avatarImageView: UIImageView!
+    @IBOutlet private var shadowView: UIView!
+    @IBOutlet private var nameLabel: UILabel!
+    @IBOutlet private var descriptionLabel: UILabel!
+
+    override func awakeFromNib() {
```

Comment:
```suggestion
// MARK: - LifeCycle
    override func awakeFromNib() {
```
+ сделай отдельный метод для тени